src/thunderbird_accounts/mail/clients.py

```diff
@@ -362,6 +369,20 @@ def update_individual(
             logging.error(f'[update_individual] err: {data}')
             raise RuntimeError(data)
 
+    def update_quota(self, principal_id: str, quota: int):
+        update_data = [
+            {'action': 'set', 'field': 'quota', 'value': quota},
+        ]
+        response = self._update_principal(principal_id, update_data)
+
+        # Returns data: null on success...
+        data = response.json()
+        error = data.get('error')
+        # I have no idea what the error is yet
+        if error:
+            logging.error(f'[update_individual] err: {data}')
+            raise RuntimeError(data)
```

Comment:
I'll cut a ticket to actually test these...This part is mostly copy paste from other update functions.